### ISSUE TITLE
BPR update

### ISSUE THREAD

**Issue by Maroyafaied** [ONRR/onrr.gov-site#959]
Understand what BPR is working on when it comes to software developed and review process .

**Comment by mcharg** [comment]
@Maroyafaied Jodie said to ask Robert Sudar, John Swedin, and Crystel Sandos. I'll send an email.

**Comment by cthomasONRR** [comment]
I had a chat with Robert Sudar to make sure I'm in the loop for tomorrow's BPR meeting. This first meeting is supposed to be very high level. The main topic for us will just be explaining that the CMS is for uploading files/information to the website and NOT for creating any type of documents. Robert knew this already. Their focus will be making sure that any automation system they create (for example- automatic pricing updates) will be compatible with the CMS.

**Comment by cthomasONRR** [comment]
The BPR meeting was from 8-10 today (9/1/21).

This "Push Data" BPR is about identifying recurring processes (for example: communication of data to the public, scheduled/standard reports, prices, and recurring company requests) that ONRR follows to develop and share solutions which communicate the agency's data to internal and/or external stakeholders. The keyword here is 'recurring' and this BPR will focus on identifying recurring processes and developing a methodology in which to automate these processes.

This first meeting was for very high-level ideas and concepts. We discussed the broad 'road map' that anything within the Push Data BPR should follow to develop a final product. Since the roadmap was not tied to a specific example, I tried to add in comments about the Data Display group when it was possible. Here are some of the topics I brought up during the meeting:
- For related statutes and regulations, I made sure they added Section 508 compliance to the list (it was previously not mentioned).
- I emphasized that while we do not handle the 'source' data, our group still does a rigorous data review, and that some form of data/peer review is needed for any process.
- I brought up that if a product created is intended to be housed on one of the public-facing websites, it needs to be compatible with the CMS. (No larger discussion about CMS, since that was too detailed for this meeting.)
- I also said that it would be nice for any download-able product from our public-facing websites to include contextual information within the downloaded file (like footnotes, metadata). I just wanted to make sure that any caveats we might have with that product aren't just housed statically on the website, but would also be with the downloaded product.
